testsuite/unittests/test_lossy_synapse.sli

```diff
@@ -0,0 +1,105 @@
+/*
+ *  test_lossy_synapse.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/* BeginDocumentation
+    Name: testsuite::test_lossy_synapse - test of lossy_synapse connection
+
+    Synopsis: (test_lossy_synapse) run -> measure average number of spikes transmitted by a lossy synapse
+
+    Description:
+    1000 spikes generated by spike_generator are sent to parrot neurons using a lossy_synapse with transmission probability 50%. 
+    If the average number of spikes detected by spike detectors equals to 500 (with marginal error 5%), the synapse works fine.
+    Also tests the interval of possible probabilities, [0, 1].
+
+    FirstVersion: June 2017
+    Author: Maximilian Schmidt, Milena Menezes Carvalho
+*/
+
+(unittest) run
+/unittest using
+
+% test parameters -------------------------------------------------
+/N_spikes  1000 def  % Number of spikes send via the lossy synapse to each parrot neuron
+/n_parrots 10 def    % Number of parrot neurons receiving spikes
```

Comment:
Initially, we were checking only one random seed with many parrot neurons to see the variability. Eventually we did implement the test across many random seeds, so indeed this could be too much at this point. Thanks for the input.